pandas/tests/series/test_operators.py

```diff
@@ -993,15 +991,44 @@ def test_operators_timedelta64_with_timedelta_invalid(self, scalar_td):
             td1 * scalar_td
         with tm.assert_raises_regex(TypeError, pattern):
             scalar_td * td1
-        with tm.assert_raises_regex(TypeError, pattern):
-            td1 // scalar_td
-        with tm.assert_raises_regex(TypeError, pattern):
-            scalar_td // td1
         with tm.assert_raises_regex(TypeError, pattern):
             scalar_td ** td1
         with tm.assert_raises_regex(TypeError, pattern):
             td1 ** scalar_td
 
+    @pytest.mark.parametrize('scalar_td', [
+        timedelta(minutes=5, seconds=4),
+        pytest.param(Timedelta('5m4s'),
+                     marks=pytest.mark.xfail(reason="Timedelta.__floordiv__ "
```

Comment:
floordiv -> rfloordiv

Reply:
?  Both `Timedelta.__floordiv__` and `Timedelta.__rfloordiv__` are wonky.  Or are you referring to something else?

Reply:
you reason is wrong, it should be rfloordiv

Reply:
You mean the `reason="Timedelta.__floordiv__`...?  The test fails because `Timedelta.__floordiv__` gets called, _not_ `Timedelta.__rfloordiv__`.